collect.py

```diff
@@ -203,6 +203,19 @@ def screenshot(driver, file_path):
     image.save(file_path)
 
 
+def get_code(driver, file_path):
+    if(not(os.path.isdir('source'))):
```

Comment:
No need to check if the directory exists, **utils.mkdir** works both if the directory already exists and if it doesn't.